sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -10,15 +10,47 @@ import { OperationOptions } from '@azure/core-http';
 import { PipelineOptions } from '@azure/core-http';
 import { RestResponse } from '@azure/core-http';
 
+// @public
+export type AppAction = "Restarted" | "Stopped" | "ChangedAppSettings" | "Started" | "Completed" | "Failed";
+
+// @public
+export interface AppConfigurationKeyValueDeletedEventData {
+    etag?: string;
+    key?: string;
+    label?: string;
+}
+
+// @public
+export interface AppConfigurationKeyValueModifiedEventData {
+    etag?: string;
+    key?: string;
+    label?: string;
+}
+
+// @public
+export interface AppEventTypeDetail {
+    action?: AppAction;
+}
+
+// @public
+export interface AppServicePlanEventTypeDetail {
+    action?: "Updated";
+    stampKind?: StampKind;
+    status?: AsyncStatus;
```

Comment:
what does an AsyncStatus of undefined mean? I think you see where I'm going with this ;)